libs/ui/components/contents/tooltip/props.ts

```diff
@@ -0,0 +1,4 @@
+import { TooltipProps } from "@chakra-ui/react";
+
+export default interface ITooltipProps extends TooltipProps {
```

Comment:
```suggestion
export interface ITooltipProps extends TooltipProps {
```

Use default for component but not for props